modules/ROOT/pages/anypoint-insights.adoc

```diff
@@ -0,0 +1,462 @@
+=== Performance Metrics and Trends
+
+*Performance Metrics* and *Performance Trends* provide insights across your deployments:
+
+* <<performance-metrics-overview>>
+* <<performance-trends-overview>>
+
+//  H4: Performance Metrics
+[[performance-metrics-overview]]
+==== Performance Metrics
+
+Total Entities::
+Total number of entities that match your currently selected time range and any query <<filters, filters>>.
+include::partial$include-insights-charts.adoc[tags="data-scope"]
+
+Entities with Errors::
+Count of entities that have an error that matches your current search conditions in the selected date and time range.
+include::partial$include-insights-charts.adoc[tags="data-scope"] 
++
+Errors are dependent on the type of deployment. 
+
+* For API implementations, an error is based on the HTTP status code (for example, `404` or `502`). 
+* For Mule apps, an error is based on the response status (for example, `FAILED`).
+
+Total Requests::
+Total request volume for entities that match your current search conditions in the selected date and time range.
+include::partial$include-insights-charts.adoc[tags="data-scope"] 
+
+Error Rate:: 
+Error rate for entities that match your current search conditions in the selected date and time range. The error rate is calculated using the total number of entity errors divided by the total number of requests for those entities.
+include::partial$include-insights-charts.adoc[tags="data-scope"]
+
+
+//
+// H4: Performance Trends
+[[performance-trends-overview]]
+==== Performance Trends
+
+Entities with Slow Response Times:: 
+Response time of the top entities that match your current search conditions in the selected date and time range. The top entities are calculated based on the percentile values of their response times (highest response time to lowest response time). The chart uses `P99` as the percentile index by default; however, you can select `P95`, `P90`, or `P75` to use as the percentile index.
+include::partial$include-insights-charts.adoc[tags="data-scope"]
+
+Error Volume:: 
+Total number of entities that match your current search conditions in the selected date and time range.
+include::partial$include-insights-charts.adoc[tags="data-scope"]
++
+Errors are dependent on the context of the entity. For API entities, an error is based on the HTTP status code (for example, `4xx` or `5xx`) and for application entities, an error is based on the response status (for example, `FAILED`)
+Request Volume:: 
+The total number of request to entities that match your current search conditions in the selected date and time range. 
+
+//
+//H3: ENTITIES LIST
+[[entities-list]]
+=== Entities List
+
+Anypoint Insights provides a list of deployed Mule apps and APIs (_entities_) that match the selected time period and any query <<filters, filters>>:
+
+Entity Name:: 
+Name of the deployed entity
+
+//TODO_TODO_TODO: Mule Gateway APIs only or other API instance?
+//TODO: SHOULD BE "API" not "Api"
```

Comment:
@sathishpaulleo 
API is always fully capitalized

Please change "Api" to "API"